contracts/ATM/ATMGovernance.sol

```diff
@@ -0,0 +1,281 @@
+pragma solidity 0.5.17;
+
+// External Libraries
+import "@openzeppelin/contracts/utils/Address.sol";
+
+// Common
+import "../util/AddressArrayLib.sol";
+import "../util/AddressLib.sol";
+
+// Contracts
+import "@openzeppelin/contracts/access/roles/SignerRole.sol";
+
+// Interfaces
+import "./IATMGovernance.sol";
+
+
+/**
+    @notice This contract is used to modify Risk Settings, CRA or DataProviders for a specific ATM.
+    @author develop@teller.finance
+ */
+contract ATMGovernance is SignerRole, IATMGovernance {
+    // TODO: Inject Settings
```

Comment:
need still?

Reply:
we will not pause this contract because it mainly contains ATM Settings so I'll remove this comment.